src/libstd/env.rs

```diff
@@ -533,10 +535,13 @@ impl Error for JoinPathsError {
 /// use std::env;
 ///
 /// match env::home_dir() {
-///     Some(path) => println!("{}", path.display()),
+///     Some(path) => println!("Your home directory, probably: {}", path.display()),
 ///     None => println!("Impossible to get your home dir!"),
 /// }
 /// ```
+#[rustc_deprecated(since = "1.28.0",
```

Comment:
Despite what the version number of the latest Nightly suggests (sorry for the confusion), 1.28 is already in beta so this should say `since = "1.29"`.

r+ with that.